_layouts/default.html

```diff
@@ -10,7 +10,7 @@
     {% if page.header_counter %} <link rel="stylesheet" href="{{site.baseurl}}/assets/css/header_counter.css"> {% endif %}
   </head>
   <body>
-      {% if page.title %} <h1 id="title"> {{ page.title }} </h1> {% endif %}
+      {% if page.show_title %} <h1 id="title"> {{ page.title }} </h1> {% endif %}
```

Comment:
Are you saying that the non empty string "page.title" is not implicitly converted to a boolean true value? Could you do this by checking the size of the string, instead of introducing a new value in the YAML section? See https://shopify.github.io/liquid/filters/size/

Reply:
It is, but that's the issue. If you look at the Contribution page, it's still showing two titles, because it's <title> is automatically generated.

Reply:
https://valeriadaneva.github.io/acle/CONTRIBUTING.html